README.txt

```diff
@@ -0,0 +1,20 @@
+# Project_sort
+sort emploee
+
+Робота програми зводиться до читання csv таблиці з даними по працівникам та очищення її по певним фільтрам.
```

Comment:
Very good. The only missing thing is the part about how to run  (you created "how it works"). Please consider english as main language for readme next time.